config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -414,6 +414,109 @@ periodics:
         - name: IGNITION_INJECT_GCE_SSH_PUBLIC_KEY_FILE
           value: "1"
 
+- name: ci-kubernetes-node-swap-conformance-ubuntu-serial
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  annotations:
+    testgrid-dashboards: sig-node-kubelet
+    testgrid-tab-name: kubelet-gce-e2e-swap-conformance-ubuntu-serial
```

Comment:
```suggestion
    testgrid-tab-name: kubelet-swap-conformance-ubuntu-serial
```

Reply:
nit mostly.

Reply:
Thanks @kannon92.

I was following the current conventions:
https://github.com/kubernetes/test-infra/blob/177cabeadc08c51f134bfdab398d71300531bf38/config/jobs/kubernetes/sig-node/node-kubelet.yaml#L365-L370

Do you think that in this case we should use something different?
Should we also change the testgrid names for existing swap serial jobs?

Reply:
https://github.com/kubernetes/test-infra/blob/177cabeadc08c51f134bfdab398d71300531bf38/config/jobs/kubernetes/sig-node/README.md

We don't really have a convention here. That link doesn't really mention gce but os, container runtime and serial.

Reply:
I'm fine to leave as is though.

Reply:
I like your testgrid name better, just wanted to make sure it's valid.

Thanks! Done :)